core/Types.h

```diff
@@ -709,6 +709,7 @@ struct DispatchArgs {
     const TypePtr &fullType;
     const TypePtr &thisType;
     const std::shared_ptr<const SendAndBlockLink> &block;
+    const Loc &originForUninitialized;
```

Comment:
Loc is by design meant to fit in a machine word and be copied.

```suggestion
    const Loc originForUninitialized;
```